app/views/registrations/new.html.erb

```diff
@@ -25,31 +25,98 @@
+    <%= f.govuk_text_field :parent_phone,
+                           label: {
+                             text: "Phone number (optional)",
+                             size: "s"
+                           },
+                           class: "nhsuk-input--width-10" %>
 
-    <%= govuk_section_break visible: true, size: "l" %>
+  <% end %>
+
+  <%= govuk_section_break visible: true, size: "l" %>
+
+  <%= f.govuk_fieldset legend: { text: "About your child", size: "l" } do %>
+
+    <%= f.govuk_text_field :first_name,
+                           label: { text: "First name", size: "s" },
+                           class: "nhsuk-input--width-20" %>
+
+    <%= f.govuk_text_field :last_name,
+                           label: { text: "Last name", size: "s" },
+                           class: "nhsuk-input--width-20" %>
+
+    <%= f.govuk_radio_buttons_fieldset :use_common_name,
+                                       legend: {
+                                         size: "s",
+                                         text: "Do they use a different name in school?"
+                                       } do %>
+      <%= f.govuk_radio_button :use_common_name,
+                               true,
+                               label: { text: "Yes" },
+                               link_errors: true do %>
+        <%= f.govuk_text_field :common_name, label: { text: "Preferred name" } %>
+      <% end %>
+      <%= f.govuk_radio_button :use_common_name, false, label: { text: 'No' } %>
+    <% end %>
+
+    <%= f.govuk_date_field :date_of_birth,
+                           date_of_birth: true,
+                           legend: { text: "Date of birth", size: "s" },
+                           hint: { text: 'For example, 27 3 2017' },
+                           link_errors: true %>
 
-    <%= f.govuk_submit "Register your interest", secondary: true %>
+    <%= f.govuk_fieldset legend: { text: "Address", size: "s" } do %>
+      <p class="nhsuk-hint">
+        If your child lives at more than one address, enter the address registered with the GP.
+      </p>
+
+      <%= f.govuk_text_field :address_line_1,
+                             label: { text: 'Address line 1' } %>
+
+      <%= f.govuk_text_field :address_line_2,
+                             label: { text: 'Address line 2 (optional)' } %>
+
+      <%= f.govuk_text_field :address_town,
+                             label: { text: 'Town or city' } %>
+
+      <%= f.govuk_text_field :address_postcode,
+                             label: { text: 'Postcode' },
+                             class: 'nhsuk-input--width-10' %>
+    <% end %>
+
+    <%= f.govuk_text_field :nhs_number,
+                           label: { text: "NHS number (optional)", size: "s" },
+                           class: "nhsuk-input--width-10" %>
   <% end %>
+
+  <%= govuk_section_break visible: true, size: "l" %>
```

Comment:
@paulrobertlloyd As discussed, I'm moving the section break out of the `fieldset`. This just has a minor effect of adding a little bit more space between the input box directly above and the `hr` generated.

The prototype puts the `hr` and following `<h1>Terms and conditions</h1>` inside the `fieldset` above.